etc/ci.sh

```diff
@@ -82,6 +82,41 @@ do
         exit 1
     else
         echo "All packages were built succesfully"
+
+        # also skip itc because it requires xgap package
```

Comment:
I was briefly confused by that "also". Perhaps:
```
#
# Now that we built packages, we try to load them.
# For this, we need to skip a few additional packages which cannot be loaded in the
# test environment.
#
# skip xgap as it can only be loaded in a GAP session started via  special helper script, *and* it requires X Window
rm -rf xgap*
# also skip itc because it requires xgap
rm -rf itc*
```